src/sentry/static/sentry/app/api.tsx

```diff
@@ -296,7 +359,7 @@ export class Client {
         success: (data, ...args) => {
           includeAllArgs ? resolve([data, ...args]) : resolve(data);
```

Comment:
There may be a bug for when `includeAllArgs` is `true` here. 🤔 

/cc @lynnagara @billyvg @markstory

Reply:
I'd love to see all the arguments being forwarded by default. Having to remember to set `includeAllArgs` and also unpack an array to get at the xhr object is more tedious than it needs to be.

Reply:
yeah -- we'd have to update the `requestPromise` calls to always expect an array